pkg/scheduling/requirements.go

```diff
@@ -220,15 +220,25 @@ func editDistance(s, t string) int {
 }
 
 func labelHint(r Requirements, key string, allowedUndefined sets.Set[string]) string {
+	splitKey := strings.Split(key, "/")
+	suffix := splitKey[1]
 	for wellKnown := range allowedUndefined {
 		if strings.Contains(wellKnown, key) || editDistance(key, wellKnown) < len(wellKnown)/5 {
 			return fmt.Sprintf(" (typo of %q?)", wellKnown)
 		}
+		if strings.Contains(wellKnown, suffix) {
```

Comment:
```suggestion
		if strings.HasSuffix(wellKnown, suffix) {
```

This is a stronger check